utils/http.go

```diff
@@ -10,13 +10,13 @@ import (
 // GetHTTPClient returns the shared HTTP client used by all Courier threads
 func GetHTTPClient() *http.Client {
 	once.Do(func() {
-		transport = http.DefaultTransport.(*http.Transport).Clone()
+		transport := http.DefaultTransport.(*http.Transport).Clone()
 		transport.MaxIdleConns = 64
 		transport.MaxIdleConnsPerHost = 8
 		transport.IdleConnTimeout = 15 * time.Second
 		client = &http.Client{
 			Transport: transport,
-			Timeout:   60 * time.Second,
+			Timeout:   30 * time.Second,
```

Comment:
this puts a 30 second limit on all outgoing requests from courier... which seems fine.. if you haven't returned in 30 seconds you're probably not going to return

Reply:
yes, 30 sec is still plenty of time